### PR TITLE
GG-25048 Resend events from agent to frontend.

### PR DESCRIPTION
**Pull request by nva**

### CODE REVIEW

File: modules/gmc-agent/src/test/java/org/gridgain/service/event/EventsServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+import static org.apache.ignite.events.EventType.EVT_NODE_LEFT;
+import static org.gridgain.agent.StompDestinationsUtils.buildEventsDest;
+import static org.mockito.Mockito.timeout;
+import static org.mockito.Mockito.verify;
+
+/**
+ * Events service test.
+ */
+public class EventsServiceTest extends AbstractServiceTest {
```
Comment (smoldachev) [inline modules/gmc-agent/src/test/java/org/gridgain/service/event/EventsServiceTest.java]: Let's add integration tests with real Ignite nodes.

File: modules/gmc-agent/src/main/java/org/gridgain/agent/StompDestinationsUtils.java [hunk 2]
```diff
@@ -46,6 +46,9 @@
     /** Metrics destination. */
     private static final String METRICS_DEST = "/app/agent/metrics/%s/add";
 
+    /** Events destination. */
+    private static final String EVENTS_DEST = "/app/agent/events/%s/add";
```
Comment (smoldachev) [inline modules/gmc-agent/src/main/java/org/gridgain/agent/StompDestinationsUtils.java]: Let's use cluster prefix: CLUSTER_PREFIX_DEST + "/events/%s/add"

File: modules/gmc-agent/src/test/java/org/gridgain/service/event/EventsServiceTest.java (hunk 1 above)
Comment (nva) [reply modules/gmc-agent/src/test/java/org/gridgain/service/event/EventsServiceTest.java]: Can you describe simple test plan?

Comment (akuznetsov-gridgain) [reply modules/gmc-agent/src/test/java/org/gridgain/service/event/EventsServiceTest.java]: @nva 
Simple plan:
1. start first node.
2. start second node -> `node joined` event should be generated.
3. stop second node -> `node leaved` event should be generated.

Comment (nva) [reply modules/gmc-agent/src/test/java/org/gridgain/service/event/EventsServiceTest.java]: Done

**Pull request merged**